dependencies.properties

```diff
@@ -26,6 +26,7 @@ version.gax_httpjson=0.75.3-SNAPSHOT
 version.com_google_protobuf=3.12.2
 version.google_java_format=1.1
 version.io_grpc=1.30.0
+version.bazel_skylib=1.0.3
```

Comment:
It is fine to have it here, but the idea is that this file contains only values shared by gradle and bazel builds. Everything which is bazel-only (like this one) can go directly to bazel files.

Reply:
Moved this back into `repositories.bzl`.